src/activity.ts

```diff
@@ -277,27 +264,32 @@ function details(
 					: dataClass.gitRepoName ?? FAKE_EMPTY
 			)
 			.replace(REPLACE_KEYS.GitBranch, dataClass.gitBranchName ?? FAKE_EMPTY);
-	}
-
-	return raw;
-}
-
-function fileDetails(_raw: string, document: TextDocument, selection: Selection) {
-	let raw = _raw.slice();
-
-	if (raw.includes(REPLACE_KEYS.TotalLines)) {
-		raw = raw.replace(REPLACE_KEYS.TotalLines, document.lineCount.toLocaleString());
-	}
-
-	if (raw.includes(REPLACE_KEYS.CurrentLine)) {
-		raw = raw.replace(REPLACE_KEYS.CurrentLine, (selection.active.line + 1).toLocaleString());
-	}
-
-	if (raw.includes(REPLACE_KEYS.CurrentColumn)) {
-		raw = raw.replace(
-			REPLACE_KEYS.CurrentColumn,
-			(selection.active.character + 1).toLocaleString()
-		);
+	} else if (_fileIcon !== undefined) {
+		// since this file is now edited with a CustomTextEditor, the following limitations apply:
+		// - unable to distinguish between debugging, viewing or editing -> editing
+		// - unable to show TotalLines, CurrentLine and CurrentColumn -> zeroed / emptied
+
+		raw = config[editing]
+			.replace(REPLACE_KEYS.TotalLines, FAKE_EMPTY)
+			.replace(REPLACE_KEYS.CurrentLine, '0')
+			.replace(REPLACE_KEYS.CurrentColumn, '0')
```

Comment:
In the case that the file is opened with a `CustomEditor`, it is impossible to obtain the current line number / column etc., so populate it with zero.